Quickref.md

```diff
@@ -182,7 +182,7 @@ For Events, the returned Event fires whenever the latest Event supplied by the w
 
 -- Similar to above, for Events.  Created Event initially tracks the first argument.
 -- At switchover, the output Event immediately tracks the new Event.
-[H]   switchPromptly    ::       Event a ->    Event (Event a)  -> m (Event a)
+[H]   switchHoldNever   ::       Event a ->    Event (Event a)  -> m (Event a)
```

Comment:
This doesn't look right - shouldn't `switchHoldNever :: Event (Event a) -> m (Event a)` ?